docs/config/README.md

```diff
@@ -446,13 +446,12 @@ it would have been `nixpkgs/pkgs`.
 
 ### Options
 
-| Variable            | Default       | Description                                                                      |
-| ------------------- | ------------- | -------------------------------------------------------------------------------- |
-| `truncation_length` | `3`           | The number of parent folders that the current directory should be truncated to.  |
-| `truncate_to_repo`  | `true`        | Whether or not to truncate to the root of the git repo that you're currently in. |
-| `prefix`            | `"in "`       | Prefix to display immediately before the directory.                              |
-| `style`             | `"bold cyan"` | The style for the module.                                                        |
-| `disabled`          | `false`       | Disables the `directory` module.                                                 |
+| Variable                    | Default                    | Description                                                                      |
```

Comment:
Variable should be changed to Option to avoid naming conflicts with format string variables.

Reply:
Changed..